worker/src/main/java/gov/cms/ab2d/worker/config/ContractToContractCoverageMapping.java

```diff
@@ -23,7 +25,15 @@ public static void init() {
         modelMapper = new ModelMapper();
         modelMapper.getConfiguration().setSkipNullEnabled(true);
 
-        Converter<Contract, ContractForCoverageDTO> coverageContractDTOConverter = new AbstractConverter<>() {
+        Converter<ContractDTO, ContractForCoverageDTO> coverageContractDTOConverter = new AbstractConverter<>() {
+
+            @Override
+            protected ContractForCoverageDTO convert(ContractDTO source) {
+                return new ContractForCoverageDTO(source.getContractNumber(), OffsetDateTime.parse(source.getAttestedOn()), ContractForCoverageDTO.ContractType.valueOf(source.getContractType().toString())); //NOSONAR
```

Comment:
This is a result of previous code debt.  Let's change the attestedOn field to be OffsetDateTime (instead of String), this will remove multiple places that directly parse the String.  If there is a need to be fancy, you can create a constructor for ContractDTO that takes a String and parses that if there is a real need.